x-pack/test/functional/services/cases/list.ts

```diff
@@ -50,6 +51,7 @@ export function CasesTableServiceProvider(
 
     async deleteCase(index: number = 0) {
       this.openRowActions(index);
+      await toasts.dismissAll();
```

Comment:
I think it would be good to change order of these two lines. Depending on where the row is on the screen, a toast could even block the button to open the rows actions. So let's first close all toasts and then open the row actions.

Reply:
https://github.com/elastic/kibana/pull/186406/commits/757f9deb70f013ae90a7acb4d99f69b858056dfb